csxtools/lmfit_models.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np

```

Comment:
@tacaswell @ericdill Should this even be in csxtools? It would seem this is very generally needed. @vivekthampy I would prefer to remove this.